src/main/java/org/folio/circulation/resources/renewal/RenewalFeeProcessingStrategy.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.circulation.resources.renewal;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.resources.context.RenewalContext;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.Result;
+
+public interface RenewalFeeProcessingStrategy {
+
```

Comment:
Personally, I'd remove this blank line, I don't think it improves the readability